python/mxnet/module/module.py

```diff
@@ -171,6 +174,77 @@ def save_checkpoint(self, prefix, epoch, save_optimizer_states=False):
             self.save_optimizer_states(state_name)
             logging.info('Saved optimizer state to \"%s\"', state_name)
 
+    def export_serving(self, prefix, epoch, signature, save_optimizer_states=False,
+                       use_synset=False):
+        """Export current progress to a zip file used by MXNet model serving.
+
+        Parameters
+        ----------
+        prefix : str
+            The file prefix to export to.
+        epoch : int
+            The current epoch number.
+        signature : dict
+            A dictionary containing model input and output information.
+            Data names or data shapes of inputs and outputs can be automatically
+            collected from module. User needs to specify inputs and outputs MIME
+            type for http request. Currently only 'image/jpeg' and 'application/json'
+            are supported. All inputs should have the same type. This also applies for
+            outputs.
+            An example signature would be:
+                signature = { "input_type": "image/jpeg", "output_type": "application/json" }
+        save_optimizer_states : bool
```

Comment:
Is this relevant for Model Serving? I think not, since model serving is about Inference only, not training. I suggest we remove this argument, unless we have a good reason to have it in.